client-runtime/src/main/java/com/microsoft/rest/v2/RestProxy.java

```diff
@@ -39,6 +36,7 @@
 import io.reactivex.exceptions.Exceptions;
 import io.reactivex.functions.Function;
 import org.joda.time.DateTime;
+import org.reactivestreams.Publisher;
```

Comment:
Is this a new dependency? I've never seen the `org.reactivestreams` package before.

Reply:
org.reactivestreams is the Reactive Streams spec that Java 9 Flows adhere to, and RxJava2 references and adheres to. The entire spec is just a few interfaces that specify a very small subset of what RxJava2 and Flowable in particular can do.

For some reason a few RxJava methods like flatMapPublisher take a Function<T, Publisher<U>>, but themselves return a Flowable<U>, instead of being called flatMapFlowable.